src/styles/utils/reset.scss

```diff
@@ -0,0 +1,14 @@
+body,
+header,
+nav,
+ul,
+p {
+  margin: 0;
+}
+
+header,
+nav,
+ul,
+p {
```

Comment:
Paddings and margins can be reset for all the styles you use and do not repeat the writing of tags.